cmd/gin-repod/main.go

```diff
@@ -269,6 +270,12 @@ Options:
 	s.SetupServiceSecret()
 	s.SetupStores()
 
+	s.Handler = handlers.CORS(
+		handlers.AllowedHeaders([]string{"Accept", "Content-Type", "Authorization"}),
```

Comment:
This is the default, sans `Origin`; is there a specific reason why this was excluded?